code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -204,7 +204,7 @@
 	STOP_PROCESSING(SSobj, src)
 
 /obj/item/radio/intercom/welder_act(mob/user, obj/item/I)
-	if(!buildstage)
+	if(buildstage != 0)

```

Comment:
can just do 
```suggestion
	if(buildstage)
```

Reply:
because if its not 0 it will be true

Reply:
You are correct, I just wrote it like that for clarity